build.gradle.kts

```diff
@@ -39,9 +39,8 @@ allprojects {
     
     tasks.withType<KotlinCompile> {
         kotlinOptions.apply {
-            languageVersion = "1.3"
+            languageVersion = "1.6"
             jvmTarget = "1.8"
-            freeCompilerArgs = freeCompilerArgs + listOf("-Xskip-runtime-version-check")
```

Comment:
1. Afaik 1.3 is not supported in 1.6 anymore, so had to bump it. Can change to `1.5` or `1.4` if there's a need for this
2. Not sure why it was initially added, but I believe `-Xskip-runtime-version-check` is not needed anymore since it's the same version of kotlin and dokka -- there's no error now. Can revert, but would love to know why it's needed then :)

Reply:
We have to stick with 1.4, otherwise Gradle 6.x won't be able to consume newer Knit (because it's bundling Kotlin 1.4). You may need `-Xsuppress-version-warnings` for that.

Regarding 2, sure, it's no longer needed